MaterialLib/MPL/CreateProperty.cpp

```diff
@@ -115,30 +115,37 @@ std::unique_ptr<MaterialPropertyLib::Property> createProperty(
             //! \ogs_file_param{properties__property__LinearProperty__reference_value}
             config.getConfigParameter<double>("reference_value");
 
-        auto const& independent_variable_config =
+        std::vector<MaterialPropertyLib::IndependentVariable> ivs;
+        auto const& independent_variables_config =
             //! \ogs_file_param{properties__property__LinearProperty__independent_variable}
```

Comment:
The documentation path has to be updated.
```suggestion
            //! \ogs_file_param{properties__property__LinearProperty__independent_variables}
```

Reply:
Fixed.